Project.toml

```diff
@@ -1,4 +1,25 @@
 name = "FiniteHorizonPOMDPs"
 uuid = "8a13bbfe-798e-11e9-2f1c-eba9ee5ef093"
-authors = ["Zachary Sunberg <sunbergzach@gmail.com>"]
+authors = ["Tomas Omasta <omastto1@fel.cvut.cz> and contributors"]
 version = "0.1.0"
+
+[deps]
+DiscreteValueIteration = "4b033969-44f6-5439-a48b-c11fa3648068"
+POMDPLinter = "f3bd98c0-eb40-45e2-9eb1-f2763262d755"
+POMDPModelTools = "08074719-1b2a-587c-a292-00f91cc44415"
+POMDPs = "a93abf59-7444-517b-a68a-c42f96afdd7d"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
```

Comment:
I don't think we need Revise as a dependency - that is a development tool.

Reply:
fixed this as well